src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -714,22 +720,31 @@ private boolean generateRecommendationsBasedOnTerms(ContainerData containerData,
                     }
 
                     RecommendationNotification recommendationNotification = null;
-                    if (isCostModel) {
+
+                    if (model.getModelName().equalsIgnoreCase(RecommendationConstants.RecommendationEngine.ModelNames.COST)) {
                         // Setting it as at least one recommendation available
                         recommendationAvailable = true;
                         recommendationNotification = new RecommendationNotification(
                                 RecommendationConstants.RecommendationNotification.INFO_COST_RECOMMENDATIONS_AVAILABLE
                         );
                     }
 
-                    if (isPerfModel) {
+                    else if (model.getModelName().equalsIgnoreCase(RecommendationConstants.RecommendationEngine.ModelNames.PERFORMANCE)) {
```

Comment:
Fix the syntax of if-else-if ladder, `else if` should start right after `if` braces